python/triton/language/extra/intel/libdevice.py

```diff
@@ -1158,6 +1316,15 @@ def erfcx(arg0, _builder=None):
         }, is_pure=True, _builder=_builder)
 
 
+@core.extern
+def erfcinv(arg0, _builder=None):
+    return core.extern_elementwise(
+        "", "", [arg0], {
+            (core.dtype("fp32"), ): ("__imf_erfcinvf", core.dtype("fp32")),
+            (core.dtype("fp64"), ): ("__imf_erfcinv", core.dtype("fp64")),
```

Comment:
```suggestion
            (core.dtype("fp64"), ): ("__imf_erfcinvd", core.dtype("fp64")),
```
Is this missing the return?

Reply:
Our bitcode library contains:
```
grep "__imf_erfcinv" *.ll                                                                               ✔  10175  18:51:49 
define weak dso_local spir_func float @__imf_erfcinvf(float noundef %0) local_unnamed_addr #11 !srcloc !270 {
define weak dso_local spir_func double @__imf_erfcinv(double noundef %0) local_unnamed_addr #12 !srcloc !561 {
```
So the version that returns a double has name `__imf_erfcinv`